ChangeLog.md

```diff
@@ -1,5 +1,290 @@
 # Changelog for haskell-language-server
 
+## 1.6.0
+
+Time for a new and exciting hls release:
+
+- It includes *three* brand new plugins:
+  - [Alternate number literals](https://haskell-language-server.readthedocs.io/en/latest/features.html#convert-numbers-to-alternative-formats) thanks to @drsooch
+  - [Qualify imported names](https://haskell-language-server.readthedocs.io/en/latest/features.html#qualify-imported-names) thanks to @eddiemundo
+  - New plugin to support [selection range](https://haskell-language-server.readthedocs.io/en/latest/features.html#selection-range) (aka double click text selection) thanks to @kokobd
+- Finally hls [supports *ghc 9.2.1*](https://github.com/haskell/haskell-language-server/issues/2179)
+  - Including core features and [many plugins](https://haskell-language-server.readthedocs.io/en/latest/supported-versions.html#plugins-support-by-ghc-version))
+  - Thanks to a great collective effort coordinated by @pepeiborra and with the help of @wz1000, @mpickering and @alanz among others
+- Hls now also [supports *ghc 9.0.2*](https://github.com/haskell/haskell-language-server/issues/297) with support for all plugins but the stylish-haskell formatter
+  - Including the [wingman plugin](https://github.com/haskell/haskell-language-server/tree/master/plugins/hls-tactics-plugin) thanks to @isovector and @anka-213
+- And many many fixes and performance improvements, thanks to all contributors!
+
+### Deprecation notice for 1.6.0
+
+- As we noted in the previous release we have dropped support for ghc versions 8.10.5 and 8.8.3 in *this release*
+  - We recommend upgrading ghc to the last minor version: 8.8.4 or 8.10.7
+  - You can read more about ghc deprecation policy and schedule [here](https://haskell-language-server.readthedocs.io/en/latest/supported-versions.html)
+- *After this release*:
+  - [We will remove all project stack.yaml's](https://github.com/haskell/haskell-language-server/issues/2533) but two: one for last lts and other for nightly. Temporary we could keep one more stack yaml when nightly upgrades the ghc version, to help in the transition
+  - [We will remove the install script](https://github.com/haskell/haskell-language-server/issues/2491) which lives [here](https://github.com/haskell/haskell-language-server/tree/master/install)
+    - If you want to install hls from source we recommend using `ghcup`. Download it and run `ghcup compile hls --help` to get more info about.
+    - Alternatively users could install hls manually following [these instructions](https://haskell-language-server.readthedocs.io/en/latest/installation.html#installation-from-source)
```

Comment:
To make this more clear, I think it would be better to add a deprecation note at the beginning of [this doc](https://haskell-language-server.readthedocs.io/en/latest/installation.html#install-specific-ghc-version).
If a user scrolls down to this next section, he/she may get confused.

Reply:
hmm, the plan would be replace installation instructions removing all references to install script. I thought do it just after the release but you are right if someone follows the link in the meanwhile it will see contradictory assertions

Reply:
i started to write the new install doc but i dont get it right easily so i think is better to remove the reference for now